app/tailwind.config.js

```diff
@@ -11,1025 +11,187 @@ const SCREENS = require('./utils/media');
-      '2/3': '66.666667%',
-      '1/4': '25%',
-      '2/4': '50%',
-      '3/4': '75%',
-      full: '100%',
-      '-1/2': '-50%',
-      '-1/3': '-33.333333%',
-      '-2/3': '-66.666667%',
-      '-1/4': '-25%',
-      '-2/4': '-50%',
-      '-3/4': '-75%',
-      '-full': '-100%',
-    }),
-    keyframes: {
-      spin: {
-        to: {
-          transform: 'rotate(360deg)',
+      animation: {
+        banner: 'banner 6s cubic-bezier(0, 0, 0.2, 1) infinite',
+      },
+      backgroundImage: {
+        'gradient-repeat-to-br': 'repeating-linear-gradient(-45deg, var(--tw-gradient-stops))',
+      },
+      colors: {
+        gray: {
+          100: '#F2F2F2',
+          200: '#E8E8E8',
+          300: '#C4C4C4',
+          400: '#A2A2A9',
+          500: '#77777F',
+          600: '#5F646E',
+          700: '#35373E',
+          800: '#151515',
+          900: '#111111',
+        },
+        red: {
+          100: '#FFBFB7',
+          200: '#FFAFA4',
+          300: '#FF9D90',
+          400: '#FF7E6C',
+          500: '#FF6D59',
+          600: '#F25C47',
+          700: '#FF472E',
+          800: '#CF2A14',
+          900: '#C21701',
+        },
+        yellow: {
+          50: '#FFF4D9',
+          100: '#FFE39D',
+          200: '#FFDD8D',
+          300: '#FFD780',
+          400: '#FFCA42',
+          500: '#FFC453',
+          600: '#FDB944',
+          700: '#FBAD35',
+          800: '#F79618',
+          900: '#F38401',
+        },
+        green: {
+          50: '#B1FEF7',
+          100: '#B1FEF7',
+          200: '#8CF9EF',
+          300: '#66F4E7',
+          400: '#03E7D1',
+          500: '#03E4CE',
+          600: '#03D6C2',
+          700: '#02BBA9',
+          800: '#02AD9D',
+          900: '#029F90',
+        },
+        pink: {
+          100: '#FFDDF6',
+          200: '#FFBAEC',
+          300: '#FF9AE4',
+          400: '#FF7CDB',
+          500: '#FF3DCA',
+          600: '#F531B3',
+          700: '#EB269D',
+          800: '#E11A87',
+          900: '#BC0166',
+        },
+        purple: {
+          200: '#E3CEFF',
+          300: '#CEA8FF',
+          400: '#C496FF',
+          500: '#AE72FF',
+          600: '#9962FD',
+          700: '#8451FC',
+          800: '#674BFD',
+          900: '#401EF8',
+        },
+        blue: {
+          50: '#D4F4FF',
+          100: '#BFF0FF',
+          200: '#99E6FF',
+          300: '#6DDBFF',
+          400: '#1EC7FF',
+          500: '#00BFFF',
+          600: '#01ADF1',
+          700: '#3787FF',
+          800: '#1B72F5',
+          900: '#015EEB',
+        },
+        primary: {
+          50: '#F1F9FE',
+          200: '#4dd2ff',
+          300: '#33ccff',
+          400: '#1ac5ff',
+          500: '#00BFFF',
+          600: '#00ace6',
+          700: '#0099cc',
+          800: '#0086b3',
+        },
+      },
+      dropShadow: {
+        blue: '1px 0px 3px #00BFFF',
+      },
+      fontFamily: {
+        heading: 'Poppins',
+        sans: [
```

Comment:
I don't think we need to set the whole list of fonts available, just add ours and let the rest of them be the default ones: https://tailwindcss.com/docs/font-family#customizing-the-default-font